pages/linux/xmount.md

```diff
@@ -1,7 +1,7 @@
 # xmount
 
 > Convert on-the-fly between multiple input and output hard disk image types with optional write cache support.
-> Creates a virtual file system using FUSE (Filesystem in Userspace) that contains a virtual representation of the input image.
+> Create a virtual file system using FUSE (Filesystem in Userspace) that contains a virtual representation of the input image.
```

Comment:
The imperative mood on both lines seems kinda weird IMO because they aren't connected, the previous way seems more natural to me, but idk, what do you guys think?

Reply:
Maybe, contains can be changed to contain here.

Reply:
@vitorhcl IG, it's fine to have the second line in imperative mood too as we specify in the style guide, we follow imperative mood for all descriptions in a page.